usaspending_api/reporting/v2/views/agencies/overview.py

```diff
@@ -1,23 +1,22 @@
 from django.db.models import Subquery, OuterRef, DecimalField, Func, F, Q, IntegerField, Value
-from django.utils.functional import cached_property
 from rest_framework.response import Response
 
 from usaspending_api.agency.v2.views.agency_base import AgencyBase, PaginationMixin
-from usaspending_api.common.helpers.fiscal_year_helpers import (
-    get_final_period_of_quarter,
-    calculate_last_completed_fiscal_quarter,
-)
 from usaspending_api.common.helpers.generic_helper import get_pagination_metadata
 from usaspending_api.references.models import ToptierAgencyPublishedDABSView, Agency
 from usaspending_api.reporting.models import ReportingAgencyOverview, ReportingAgencyTas, ReportingAgencyMissingTas
 from usaspending_api.submissions.models import SubmissionAttributes
 
 
-class AgenciesOverview(AgencyBase, PaginationMixin):
+class AgenciesOverview(PaginationMixin, AgencyBase):
```

Comment:
Are these swapped to support the call to `super()` below? If so, does `PaginationMixin` also need to be initialized?